src/api/procedures/createGroup.ts

```diff
@@ -80,20 +80,17 @@ export async function prepareCreateGroup(
   const rawTicker = stringToTicker(ticker, context);
   const { transactions } = permissionsLikeToPermissions(permissions, context);
 
-  const groups = await token.permissions.getGroups();
+  const { custom } = await token.permissions.getGroups();
```

Comment:
I think we should check known permissions as well. If the user tries to re-create the `Full` group, we should warn him that it already exists. In fact, the error data should contain the existing group ID (`groupId: PermissionGroupType | BigNumber`)